### PR TITLE
added approval flash script.

### PR DESCRIPTION
**Pull request by bthaile**
simple script to approve an account.  pass in user address, if not then use existing env. variable for private key. I thought this would be easier to approve multiple accounts via a command line.

### CODE REVIEW

File: scripts/flash/approve-account.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  approveAugurEternalApprovalValue(augur, address, auth, function (err) {
+    if (err) {
+      console.log(chalk.red(err));
+      return callback(JSON.stringify(err));
+    }
+    callback(null);
+  });
+}
+
+function help(callback) {
```
Comment (tinybike) [inline scripts/flash/approve-account.js]: Why does `help` require a `callback` argument?  It does not perform any asynchronous operations.

File: scripts/flash/approve-account.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  console.log(chalk.red("account address is needed to get approval"));
+  callback(null);
+}
+
+function approveAccount(augur, params, auth, callback) {
+  if (params === "help") {
+    help(callback);
+  } else {
+    var account = params;
+    if (account === undefined) {
```
Comment (tinybike) [inline scripts/flash/approve-account.js]: Suggest `if (account == null) {` instead to handle both `null` and `undefined` inputs.

File: scripts/flash/approve-account.js (hunk 1 above)
Comment (bthaile) [reply scripts/flash/approve-account.js]: the call to the script is passed to help() a callback needs to be called to tell index.js the script is done running. index.js doesn't know help is called it's just executing the script.

File: scripts/flash/approve-account.js (hunk 2 above)
Comment (bthaile) [reply scripts/flash/approve-account.js]: right, forgot that one. more convenient. Will make change.

**Pull request merged**